.travis.yml

```diff
@@ -191,7 +191,7 @@ script:
     # - echo "$LD_LIBRARY_PATH"
     # - echo "$DYLD_FALLBACK_LIBRARY_PATH"
   - python3 -m helics
-  - python3 -m pytest -v ../tests/python_helics
+  - python3 -m pytest -sv ../tests/python_helics
```

Comment:
Do we want this to always show captured stdout and stderr? It could make the tests verbose.

Reply:
at least until we fully resolve the issues with the python tests,  doesn't seem to output much right now